### PR TITLE
Add getUrl and allow user to specify session capabilities

### PR DESCRIPTION
**Pull request by vabresto**
Thanks for this great library! Using it for a personal project and found myself needing to get the current URL as well as needing to change the session capabilities, and I wanted to contribute back to the source.

### CODE REVIEW

File: src/webdriver.nim [hunk 1]
```diff
@@ -33,6 +33,9 @@ type
   ProtocolException* = object of WebDriverException
   JavascriptException* = object of WebDriverException
 
+let
+  defaultSessionCapabilities = %*{"capabilities": {"browserName": "firefox"}}
+
```
Comment (vabresto) [inline src/webdriver.nim]: I wanted to make this `const` but the compiler doesn't seem to like a `const JsonNode`, so I used `let` instead. If there's a way to make this `const`, let me know; that seems more appropriate.

Comment (dom96) [reply src/webdriver.nim]: LGTM. Only thing I can suggest is to export this so that it's shown in the docgen.

**Review by dom96 — APPROVED** [review]

File: src/webdriver.nim (hunk 1 above)
Comment (vabresto) [reply src/webdriver.nim]: Do you mean mark it public? Do you want me to add some kind of additional comments to it?

Comment (dom96) [reply src/webdriver.nim]: nah, feel free to just stick a `*` in there :)

**Pull request merged**